### PR TITLE
AGENT-864: Presubmit job to test adding nodes to a cluster

### PR DESCRIPTION
**Pull request by rwsu**
The job installs a compact cluser and configures two extra worker nodes. The extra worker nodes are added to the cluster after the cluster has been installed.

Depends on: https://github.com/openshift-metal3/dev-scripts/pull/1654

### CODE REVIEW

File: ci-operator/config/openshift/installer/openshift-installer-master.yaml [hunk 1]
```diff
@@ -781,6 +781,21 @@ tests:
     cluster_profile: packet-assisted
     workflow: agent-e2e-appliance-ipv4
   timeout: 6h0m0s
+- as: e2e-agent-compact-ipv4-add-nodes
+  cluster: build05
```
Comment (andfasano) [inline ci-operator/config/openshift/installer/openshift-installer-master.yaml:785]: Probably for the first period I'd better to have it optional, ie: 

```
optional: true
```

Comment (rwsu) [reply ci-operator/config/openshift/installer/openshift-installer-master.yaml:785]: Done.

**Comment by pawanpinjarkar** [comment]
@rwsu looks like the dev-scripts needs a fix ? 
```
+(./agent/07_agent_add_node.sh:42): sudo virt-xml ostest_extraworker_0 --add-device --disk ./node.x86_64.iso,device=cdrom,target.dev=sdc
ERROR    Error: --disk ./node.x86_64.iso,device=cdrom,target.dev=sdc: Size must be specified for non existent volume 'node.x86_64.iso'
```

**Comment by rwsu** [comment]
> /pj-rehearse

This was addressed in https://github.com/openshift-metal3/dev-scripts/pull/1685

**Comment by pawanpinjarkar** [comment]
/pj-rehearse pull-ci-openshift-installer-master-e2e-agent-compact-ipv4-add-nodes

**Review by pawanpinjarkar — COMMENTED** [review]
/lgtm

**Pull request merged**